### PR TITLE
adding my data in persons.js

### PR DESCRIPTION
**Pull request by axayjha**
Adding my profile data

### CODE REVIEW

File: src/assets/persons.js [hunk 1]
```diff
@@ -15202,22 +15202,6 @@ export const data = {
         country: "India",
       },
     },
-    {
-      id: uuidv4(),
```
Comment (MattCSmith) [inline src/assets/persons.js:15206]: This pull request will remove Akshay from the person.js file. Please address this

**Review by MattCSmith — CHANGES_REQUESTED** [review]
Please ensure this PR does not remove people from teh persons.js file.

File: src/assets/persons.js (hunk 1 above)
Comment (axayjha) [reply src/assets/persons.js:15206]: I am intentionally removing it, because its a duplicate. The same data exists around line 20730.

**Comment by axayjha** [comment]
I was removing a completely duplicate entry. (the same profile is present around line 20730). If you wish, I can restore it back.

**Comment by MattCSmith** [comment]
Ah no thats perfect. Might be worth mentioning reasoning behind things like that in the pull request notes. 👍

**Comment by axayjha** [comment]
Agreed, will keep in mind. Thanks

**Pull request merged**